cypress/integration/business-pages/business-pages.test.js

```diff
@@ -22,7 +23,8 @@ describe('Business pages test ', () => {
     cy.stubRequest('getAllBusinessTexts', getAllBusinessTexts).as(
       'getAllBusinessTexts'
     );
-    cy.visit('/business-pages');
+    cy.visit('/');
```

Comment:
maybe make sense to receive path from configs

Reply:
We don't have home page on '/' route. When we merge pr with homepage I will fix it